client/my-sites/plans/jetpack-plans/style.scss

```
@@ -4,78 +4,24 @@
 /* cyrillic-ext */
 @font-face {
 	font-family: Inter;
-	font-style: normal;
-	font-weight: 400;
-	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-Regular.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-Regular.woff?v=3.19) format("woff");
-}
-@font-face {
-	font-family: Inter;
-	font-style: italic;
-	font-weight: 400;
-	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-Italic.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-Italic.woff?v=3.19) format("woff");
-}
-
-@font-face {
-	font-family: Inter;
-	font-style: normal;
-	font-weight: 500;
-	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-Medium.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-Medium.woff?v=3.19) format("woff");
-}
-@font-face {
-	font-family: Inter;
-	font-style: italic;
-	font-weight: 500;
+	font-weight: 100 900;
 	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-MediumItalic.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-MediumItalic.woff?v=3.19) format("woff");
-}
-
-@font-face {
-	font-family: Inter;
 	font-style: normal;
-	font-weight: 600;
-	font-display: swap;
-	src:
-		url(https://s0.wp.com/i/fonts/inter/Inter-SemiBold.woff2?v=3.19) format("woff2"),
-		url(https://s0.wp.com/i/fonts/inter/Inter-SemiBold.woff?v=3.19) format("woff");
-}
-@font-face {
-	font-family: Inter;
-	font-style: italic;
-	font-weight: 600;
-	font-display: swap;
+	font-named-instance: "Regular";
```

And this line is throwing a stylelint error too, causing the "Code style (Web app)" check to fail. You could disable the rule here too:

```suggestion
	/* stylelint-disable-next-line property-no-unknown */
	font-named-instance: "Regular";
```

---

Also added comments to both of these properties.